### PR TITLE
use_global_stats 

### PR DESCRIPTION
**Pull request by sunrongda**
## Description ##
(Brief description on what this PR is about)
''''''''''''
use_global_stats (boolean, optional, default=0) – Whether use global moving statistics instead of local batch-norm. This will force change batch-norm into a scale shift operator

'''''''''
When use_global_stats is true, the parameters of this layer remain unchanged, which should be applied in the case of test, while in the process of training, make sure that use_global_stats is not true, of course, use_global_stats is false by default.

''''''''
When I look at faster rcnn, why is his BN layer use_global_stats all True? It's supposed to be False when I train.

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] The PR title starts with [MXNET-$JIRA_ID], where $JIRA_ID refers to the relevant [JIRA issue](https://issues.apache.org/jira/projects/MXNET/issues) created (except PRs with tiny changes)
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- Unit tests are added for small changes to verify correctness (e.g. adding a new operator)
- Nightly tests are added for complicated/long-running ones (e.g. changing distributed kvstore)
- Build tests will be added for build configuration changes (e.g. adding a new build option with NCCL)
- [ ] Code is well-documented: 
- For user-facing API changes, API doc string has been updated. 
- For new C++ functions in header files, their functionalities and arguments are documented. 
- For new examples, README.md is added to explain the what the example does, the source of the dataset, expected performance on test set and reference to the original paper if applicable
- Check the API doc at http://mxnet-ci-doc.s3-accelerate.dualstack.amazonaws.com/PR-$PR_ID/$BUILD_ID/index.html
- [ ] To the my best knowledge, examples are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, API doc)
- [ ] Feature2, tests, (and when applicable, API doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made.
- Interesting edge cases to note here

### CODE REVIEW

**Comment by wkcn** [comment]
Since the batch size is usually small in Object Detection, such as 1 image per GPU, we don’t need to update the mean and variance of Batch Normalization.

**Comment by piyushghai** [comment]
@sunrongda CI failed for an unrelated reason. Can you push an empty commit again to re-trigger the CI ? 

@mxnet-label-bot Add [Example, pr-awaiting-review]

**Comment by sunrongda** [comment]
> Since the batch size is usually small in Object Detection, such as 1 image per GPU, we don’t need to update the mean and variance of Batch Normalization. We update BN when the total batch size >= 64.

tI know what's going on. Thank you.Please close it.

**Comment by wkcn** [comment]
Close it because the issue has been addressed.
Thank you for the discussion! Feel free to reopen it.
